.circleci/config.yml

```diff
@@ -0,0 +1,95 @@
+---
+version: 2
+
+references:
+  images:
+    middleman: &MIDDLEMAN_IMAGE hashicorp/middleman-hashicorp:0.3.41
+
+  cache:
+    rubygem: &RUBYGEM_CACHE_KEY static-site-gems-v1-{{ checksum "content/Gemfile.lock" }}
+
+jobs:
+  deploy-website:
+    docker:
+      - image: *MIDDLEMAN_IMAGE
+    environment:
+      NO_WARM: true # we don't warm the cache to parallelize wgets in CircleCI
+    steps:
+      - checkout
+
+      # pull and update git submodules
+      - run: make sync
+
+      # restores gem cache
+      - restore_cache:
+          key: *RUBYGEM_CACHE_KEY
+
+      - run:
+          name: install gems
+          working_directory: content
+          command: bundle check || bundle install --path vendor/bundle --retry=3
+
+      # saves gem cache if we have changed the Gemfile
+      - save_cache:
+          key: *RUBYGEM_CACHE_KEY
+          paths:
+            - content/vendor/bundle
+
+      - run:
+          name: middleman build
+          working_directory: content
+          command: bundle exec middleman build
+
+      - run: ./content/scripts/deploy.sh
+
+  warm-cache-provider-aws:
+    docker:
+      - image: circleci/golang:latest
+    steps:
+      - run:
+          name: website deploy
```

Comment:
What's the significance of these `name` fields? It doesn't seem to match the content, and it doesn't seem to show up anywhere I can find it in the Circle interface.